### ISSUE TITLE
Downloading of icons does not work

### ISSUE THREAD

**Issue by dorianim** [linuxmuster/linuxmuster-linbo7#84]
The feature proposed in #83 and implemented in 4afe1e4 does not seem to be working correctly.
My `/srv/linbo/icons`:
![image](https://user-images.githubusercontent.com/30153207/193913240-82fc036d-a121-4e8b-a08d-12e84198fe63.png)
However, on the client:
![image](https://user-images.githubusercontent.com/30153207/193913300-0f1115c7-d0ef-45a4-b2e4-4311c541f28c.png)

**Comment by HappyBasher** [comment]
It's the code from PR #184: https://github.com/linuxmuster/linuxmuster-linbo/commit/5f6abff29474d4e0e464ade5bd1c293e59ed8158
I never tested this, I don't know anything about gui theming.

**Comment by dorianim** [comment]
I looked into it: You have changed the environment (everything is downloaded to `/cache/icons` instead of `/icons`) and did not properly adapt the code from the old pr.
https://github.com/linuxmuster/linuxmuster-linbo7/blob/4afe1e4b00ab38e4b91336e6d258f0436fa82183/linbofs/usr/bin/linbo_cmd#L2941

This line has to be changed to

```bash
 linbo_gui_theme_icons=$(sed -e '1,/^\[icons\]/ d' /cache/icons/$linbo_gui_themefile | sed -e '/^\[.*\]/q' | sed '$d' | sed '/^$/d' | awk -F\= '{ print $2 }' | awk '{ print $1 }') 
```
and it works fine.

**Comment by HappyBasher** [comment]
Thanks for the contribution. Hopefully finally fixed in [v4.0.40](https://github.com/linuxmuster/linuxmuster-linbo7/compare/v4.0.39...v4.0.40).

**Comment by dorianim** [comment]
Can confirm this works in [v4.0.41-0](https://github.com/linuxmuster/linuxmuster-linbo7/releases/tag/v4.0.41)
